### ISSUE TITLE
[Project]

### ISSUE THREAD

**Issue by matteo-d-m** [brainhackrome/brainhackrome.github.io#3]
## Project info

**Title:**
I Know What You Will Do: Forecasting Motor Behaviour from EEG Time Series

**Project lead:**
Matteo De Matola <sup>[1](https://webapps.unitn.it/du/en/Persona/PER0247884/Pubblicazioni),[2](https://github.com/matteo-d-m)</sup>, Anna Notaro

**Project collaborators:**
We are actively looking for collaborators! If you are interested, talk to us at the event or write us before:
- matteo [dot] dematola [at] unitn [dot] it
- anna [dot] notaro [at] studbocconi [dot] it

**Project Description:**

This project aims at developing 

**Data to use:**
We will use the WAY-EEG-GAL dataset, which is described in the original paper by [Luciw et al. (2014)](https://www.nature.com/articles/sdata201447) and is publicly available on [figshare](https://figshare.com/collections/WAY_EEG_GAL_Multi_channel_EEG_Recordings_During_3_936_Grasp_and_Lift_Trials_with_Varying_Weight_and_Friction/988376).

Contributors will be provided with a dataset version optimised for machine learning.

**Link to project repository/sources:**

**Goals:**
Forecast 

**Good first issues:**

1. Load the dataset using [MNE](https://mne.tools/stable/index.html) and custom Python scripts
2. Implement a deep learning algorithm that can forecast multivariate time series with satisfying accuracy (e.g., [WaveNet](https://arxiv.org/abs/1609.03499))
3. Train and the test the model on our dataset

**Skills:**

**Tools/Software/Methods to Use:**

The official language of this project is Python. Contributors will need a laptop with at least:

- A working Python installation
- NumPy
- Matplotlib
- MNE
- scikit-learn
- PyTorch 
- A modern code editor like VSCode

Participants without a functioning Python ecosystem can follow the installation instructions at this link, possibly before the event. On the first day of the event

**Communication channels:**

 

**Project labels**

- [ ] I added all of the labels I want an associate to my project

### Submission checklist

*Once the issue is submitted, please check items in this list as you add under ‘Additional project info’*

- [ ] Link to your project: could be a code repository, a shared document, etc.
- [ ] Goals: describe what you want to achieve during this brainhack.
- [ ] Flesh out at least 2 “good first issues”: those are tasks that do not require any prior knowledge about your project, could be defined as issues in a GitHub repository, or in a shared document.
- [ ] Skills: list skills that would be particularly suitable for your project. We ask you to include at least one non-coding skill. Use the issue labels for this purpose.
- [ ] Chat channel: A link to a chat channel that will be used during the event. This can be an existing channel or a new one. We recommend using the [Brainhack space on Mattermost](https://mattermost.brainhack.org/).

Optionally, you can also include information about:

- [ ] Number of participants required.
- [ ] Twitter-sized summary of your project pitch.
- [ ] Provide an image of your project for the website. 

We would like to think about how you will credit and onboard new members to your project. If you’d like to share your thoughts with future project participants, you can include information about:

- [ ] Specify how you will acknowledge contributions (e.g. listing members on a contributing page).
- [ ] Provide links to onboarding documents if you have some:

**Comment by matteo-d-m** [comment]
Hi @brainhackrome my project is ready! :wave: :rocket:
